08_deploy_bmo.sh

```diff
@@ -2,15 +2,39 @@
 
 eval "$(go env)"
 
+source common.sh
+
 # Get the latest bits for baremetal-operator
 export BMOPATH="$GOPATH/src/github.com/metalkube/baremetal-operator"
 
-oc --as system:admin --config ocp/auth/kubeconfig new-project bmo-project
+# Make a local copy of the baremetal-operator code to make changes
+cp -r $BMOPATH/deploy ocp/.
+sed -i 's/bmo-project/openshift-machine-api/g' ocp/deploy/role_binding.yaml
```

Comment:
I think this is supposed to be handled via https://github.com/openshift-metalkube/dev-scripts/pull/175 right?  So maybe we should rebase this patch on that one from @sadasu ?

Reply:
Yes you're right, didn't see that until now.  Also I don't think the copy is a good idea.. we need to fix up the baremetal-operator stuff